### DISCUSSION TITLE
Добавить больше оружия(что?)

### DISCUSSION THREAD

**Discussion by Starikprostoy** [octothorp-team/public-dobrograd#666] [Предложения]
Вообщем, так как мне нехер делать я предлагаю добавить немного новых пушек. Например двухстволка(ТОЗ-34 или Вертикалка уже сами решайте) на основе Annabel из Халвы, SPAS-12 на основе СПАСа из Халвы, лом/монтировка на основе монтировки из Халвы, кастеты(как?) ну и метательное для граждан(да).

**Comment by LENFYS** [comment]
Против

**Comment by travka4000** [comment]
делать нечего, какие спасы 12 какие двухстволки, ты еще предложи добавить место ксс м9к пак

**Reply by travka4000** [reply]
против

**Comment by SnakeLuck** [comment]
Против. Что это за БРЕД?
